scss/input/_layout.scss

```diff
@@ -191,6 +199,122 @@
         }
     }
 
+    // Forms
+    $form-spacer: $padding-x * 2 !default;
+
+    .k-form,
+    .k-form-inline {
+        font-size: $font-size;
+        line-height: $line-height;
+        padding: $form-spacer;
+
+        fieldset {
```

Comment:
It appears that **.k-fieldset** has been previously defined, but not documented. **.k-form fieldset** follows the idea for not using additional classes for styling form elements